javascript/javascript2/week1/danishCharacters.js

```diff
@@ -0,0 +1,18 @@
+const danishString = "Jeg har en blå bil";
+const danishString2 = "Blå grød med røde bær";
+const myObj = {};
+
+
+// Function for returning Danish letters from a string
+function getDanishCharacters(words) {
+    let arr = words.match(/[^\x00-\x7F]+/gi); //Not including ASCII character set (0 - 127)
+    myObj.total = arr.length;
+    arr.forEach(element => {
+        myObj[element] = (myObj[element] || 0) + 1;
+    })
+
+    return myObj;
+}
```

Comment:
Naming is one of the most difficult things in programming, because good naming makes your code much easier to read and understand!

When you name things like functions and variables, try to be as explicit as possible in what the function or variable represents.

For instance, the variables `arr`, `myObj` and `element` do tell me something about what's inside the variables, but I think it can be more specific and descriptive than that. Give it a shot :) 

And remember that long names are OK. Of course you don't want to make them longer than necessary, but It's much better to have a long variable or function name that's easy to understand than a short one that is hard to understand.